src/libraries/Native/Unix/System.Security.Cryptography.Native.Android/pal_sslstream.c

```diff
@@ -652,6 +652,12 @@ void AndroidCryptoNative_SSLStreamRelease(SSLStream* sslStream)
 
 int32_t AndroidCryptoNative_SSLStreamGetApplicationProtocol(SSLStream* sslStream, uint8_t* out, int32_t* outLen)
 {
+    if (g_SSLEngineGetApplicationProtocol == NULL)
+    {
+        LOG_ERROR ("SSLStreamGetApplicationProtocol is only supported from API level 29 and above");
+        return FAIL;
```

Comment:
how is FAIL handled in the managed layer? We need to do something gracefully. If it's already handled gracefully then we shouldn't print `LOG_ERROR` since it's an "expected" situation.

Reply:
It is handled gracefully. Essentially it transforms into `SslStreamPal.GetNegotiatedApplicationProtocol` returning `null` which handled here:

https://github.com/dotnet/runtime/blob/01b7e73cd378145264a7cb7a09365b41ed42b240/src/libraries/System.Net.Security/src/System/Net/Security/SecureChannel.cs#L845-L846

Reply:
Ok good, we should replace the `LOG_ERROR` with just a comment then since it's not an error scenario:

```
// SSLEngine.getApplicationProtocol() is only supported from API level 29 and above
```